nextjs/packages/next-env/index.ts

```diff
@@ -67,15 +67,16 @@ export function processEnv(
 
 export function loadEnvConfig(
   dir: string = process.cwd(),
-  dev?: boolean,
-  log: Log = console
+  log: Log = console,
+  ignoreCache = false
```

Comment:
Why did you remove `dev`?

Reply:
So `ignoreCache` is what fixes the issue right? Makes sense.

It would be slightly better to add an object as the new param as `{ignoreCache}`. I always fine unnamed function params error prone, especially when we are in a fork situation.

Reply:
> Why did you remove dev?

Not used in the function body

Reply:
> Not used in the function body

Normally I would agree, but in the context of a fork, this makes upstream merges more difficult